### DISCUSSION TITLE
cancer tumor cell GO slim wanted to generate relevant pathways

### DISCUSSION THREAD

**Discussion by nathandunn** [geneontology/helpdesk#342] [Ideas]
Hello,

Our group is trying to find more a GO slim that can be used to find pathways for cancer tumor cells in humans. 

https://xenabrowser.net/ / https://github.com/ucscXena/XenaGoWidget/ 

I haven't seen anything explicit, but maybe you could point me in the right direction. 

Cross-reference here: https://github.com/ucscXena/XenaGoWidget/issues/604

Thanks,

Nathan

**Comment by nathandunn** [comment]
FYI @lpalbou

**Comment by suzialeksander** [comment]
Hi @nathandunn,

We're collecting use cases for Slims, so I will add this to the list. 

Mentioning https://github.com/geneontology/amigo/issues/144

**Comment by cmungall** [comment]
I'm working with Tom H on aligning PW to GO. This is a category of slim that could be autogenerated based on presence/absence of a mapping.

reaction slim: Rhea (better than EC as Rhea gives level of specificity)
pathway slim: PW or Reactome (PW may be better for a pathway via as Reactome maps to different levels in GO) or metacyc (but metacyc mappings are very incomplete)

**Comment by nathandunn** [comment]
@cmungall  to be able to push the autogenerated slims to our tool would be a nice feature

**Comment by nathandunn** [comment]
FYI @lpalbou @jingchunzhu

**Comment by lpalbou** [comment]
@cmungall turns out Suzi created a GO slim for the Xena Widget / TCGA data: https://github.com/ucscXena/XenaGoWidget/blob/develop/src/data/genesets/tgac.js

The gene sets are obsolete (2y+) but the slim set could be valid. Who would be our expert in GO to ask for more feedback ? It would be great if this could be an official GO slim for cancer.

**Comment by ValWood** [comment]
I'm not sure what you want in a GO slim for cancer. A good slim covering human should do this job as cancer is not restricted to a small subset of genes or processes (covers cytoskeletal processes, DNA. metabolism, gene expression, mRNA metabolism, cell cycle control, vesicle-mediated transport, telomeric processes etc. etc.)
If you can provide more details about what you need in a "cancer slim", we can see how well the new generic slim covering human fits (you are likely to need a subset of this). However,  it's  a moving target as annotations to cancer-causing genes become extended (i.e there are very few cell level processes that are not implicated in cancers, so maybe what you need is a good cell-level slim?)

@RLovering @pgaudet

**Comment by suzialeksander** [comment]
@nathandunn we are doing a Slim workshop at this GOC meeting this week, scheduled for Thursday's second session.  Let me know if you or your group would like to attend and I can send Zoom/agenda info.

**Comment by ValWood** [comment]
For cancer slim, this might be a good point of reference:

This is pombe cancer gene orthologs slimmed with the pombe process slim (which is basically the cell level component of the new generic slim, give or take a few terms).

https://www.pombase.org/slim:bp_goslim_pombe/from/id/57916b05-9c2c-4669-97ab-a39d49b409d1

(this shows that cancer genes are distributed  is across most cell-level processes)
